Microsoft.Toolkit.Parsers/Markdown/Helpers/LineInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+namespace Microsoft.Toolkit.Parsers.Markdown.Helpers
+{
+    internal class LineInfo
```

Comment:
Is this for tracking block positions?

If so, could be useful to expose for locating/highlighting text based on parsing?

Reply:
Again, this should be work for a PR based #1767, where we open up the internals of the Parser for tinkering.
 
Since you couldn't really do much with this info right now, and it would require considerable more work to expose, leave it for now.

This PR is big enough as it is :)